File: app/src/main/java/com/legue/axel/lolsummonertool/network/response/champion/PassiveResponse.java
package com.legue.axel.lolsummonertool.network.response.champion;

import com.legue.axel.lolsummonertool.database.model.champion.PassiveImage;

public class PassiveResponse {

    public String name;
    public String description;
    public PassiveImage image;

    public PassiveResponse() {
    }

    public PassiveResponse(String name, String description, PassiveImage image) {
        this.name = name;
        this.description = description;
        this.image = image;
    }
}

File: app/src/main/java/com/legue/axel/lolsummonertool/network/response/champion/PassiveResponse.kt
package com.legue.axel.lolsummonertool.network.response.champion

import com.legue.axel.lolsummonertool.database.model.champion.PassiveImage

data class PassiveResponse(
        var name: String,
        var description: String,
        var image: PassiveImage
)
